### PR TITLE
883 about page for luigi website

### PR DESCRIPTION
**Pull request by marynaKhromova**

### CODE REVIEW

File: website/landingpage/public/index.html [hunk 1]
```diff
@@ -32,6 +32,9 @@
               class="close-btn hide-for-xlarge"
             ></button>
             <ul class="menu">
+              <li class="item">
+                <a href="/about" class="">About Luigi</a>
```
Comment (pekura) [inline website/landingpage/public/index.html]: This link, at least locally running gulp, is not valid:
Bildschirmfoto 2019-10-23 um 09 50 33

File: website/landingpage/dev/src/pages/about.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    <div class="column small-12 large-6">
+      <div class="sub-title has-number"><span class="text">Introducing Luigi</span></div>
+      <div class="title-2">
+        A better answer to distributed UI development - secure and infinitely scalable.
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="columns xlarge-9 content-wrapper" id="stickyAnchor" data-something>
+      <p>Offering a wide array of products, many companies are also offering just as many entry points as user interfaces. Oftentimes, these entry points and user interfaces lack in consistency and recognizability. Seamless transition between products is often hard to achieve, and this is one of the cornerstones of a great customer experience. Ensuring a consistent look and feel and harmonizing all the products and applications is a demanding and an expensive task. It means rebuilding them from the ground up and accommodating to particular UX requirements. One of the reasons why it is so difficult is that different products are developed on different technology stacks.</p>
```
Comment (alexandra-simeonova) [inline website/landingpage/dev/src/pages/about.html]:
```suggestion
      <p>Offering a wide array of products, many companies are also offering just as many entry points as user interfaces. Oftentimes, these entry points and user interfaces lack in consistency and recognizability. Seamless transition between products is often hard to achieve, and this is one of the cornerstones of a great customer experience. Ensuring a consistent look and feel and harmonizing all the products and applications is a demanding and expensive task. It means rebuilding them from the ground up and fulfilling particular UX requirements. One of the reasons why it is so difficult is that different products are developed on different technology stacks.</p>
```

File: website/landingpage/dev/src/pages/about.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+      <p>We aspire to sell our customers a great customer experience and we believe that a great customer experience should be available to both: customers and to the people who make this customer experience possible – administrators and developers.</p>
```
Comment (alexandra-simeonova) [inline website/landingpage/dev/src/pages/about.html]:
```suggestion
      <p>We aspire to sell our customers a great customer experience and we believe that a great customer experience should be available to both customers and the people who make this customer experience possible – administrators and developers.</p>
```

File: website/landingpage/dev/src/pages/about.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+      <p>With this goal in mind, we have designed Luigi which makes it possible to transform your product smoothly from a monolith to a micro-service and micro-frontend architecture. With Luigi, you build an overarching administration and business user application and re-use the functions and views of the applications currently available. Also, you can take modules or even entire applications from third-party vendors and integrate them directly into your application.</p>
```
Comment (alexandra-simeonova) [inline website/landingpage/dev/src/pages/about.html]:
```suggestion
      <p>With this goal in mind, we have designed Luigi, which makes it possible to transform your product smoothly from a monolith to a micro-service and micro-frontend architecture. With Luigi, you build an overarching administration and business user application and re-use the functions and views of the applications currently available. Also, you can take modules or even entire applications from third-party vendors and integrate them directly into your application.</p>
```

File: website/landingpage/dev/src/pages/about.html [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+      <p>Luigi is your choice, if you are using independent distributed UI teams to develop, deploy, maintain, and operate modules and end-to-end features independently. Such distributed teams do not have to make commitments to the same repository and depend on each other during the development and release cycles. This way, there is no need for any specific team to be the domain expert and fully own the entire underlying feature set and product portfolio behind the monolith UI. Another advantage is that, once developed, these independently developed features can be used by multiple solutions.</p>
```
Comment (alexandra-simeonova) [inline website/landingpage/dev/src/pages/about.html]:
```suggestion
      <p>Luigi is your choice if you are using independent distributed UI teams to develop, deploy, maintain, and operate modules and end-to-end features independently. Such distributed teams do not have to make commitments to the same repository and depend on each other during the development and release cycles. This way, there is no need for any specific team to be the domain expert and fully own the entire underlying feature set and product portfolio behind the monolith UI. Another advantage is that, once developed, these independently developed features can be used by multiple solutions.</p>
```

File: website/landingpage/dev/src/pages/about.html [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+      <p>Finally, Luigi provides out-of-the-box a shell bar as well as a navigation component and easy-to-use functions to display alerts, confirmation messages, etc.</p>
```
Comment (alexandra-simeonova) [inline website/landingpage/dev/src/pages/about.html]:
```suggestion
      <p>Finally, Luigi provides out-of-the-box features such as a shell bar, a navigation component, and easy-to-use functions to display alerts, confirmation messages, etc.</p>
```

In German, adjectives like "out-of-the-box" can be added to the beginning of the sentence, but it sounds awkward in English. Normally you would say "Luigi provides {function-name} out of the box." or "Luigi provides an out-of-the-box {function-name}." or "Luigi provides out-of-the-box functions such as {function-name}."

File: website/landingpage/dev/src/pages/about.html (hunk 2 above)
Comment (alexandra-simeonova) [reply website/landingpage/dev/src/pages/about.html]: I added some language comments but feel free to ignore them/consult with someone else first 🙂

File: website/landingpage/public/index.html (hunk 1 above)
Comment (marynaKhromova) [reply website/landingpage/public/index.html]: Thank you for review, the bug will be fixed within the follow-up ticket https://github.com/SAP/luigi/issues/921

**Pull request merged**